ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -152,8 +152,8 @@ else
 	then
 		if param compare SYS_AUTOCONFIG 1
 		then
-			# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal, next flight UUID
-			param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO* COM_FLIGHT_UUID
+			# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal, mag cal, next flight UUID
+			param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO* CAL_MAG* COM_FLIGHT_UUID
```

Comment:
Can you skip this change and we'll sort it out separately in https://github.com/PX4/Firmware/pull/13276?